sort-python-dictionary/sorting_a_dictionary.py

```diff
@@ -0,0 +1,56 @@
+# Sorting lists with the sorted() function
```

Comment:
I'm not sure about the value those code snippets bring as they're thrown in without much context.

Reply:
I've changed the comment to

> # Reviewing the sorted() function by sorting a list